.github/workflows/master.yml

```diff
@@ -0,0 +1,55 @@
+name: Master Build
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: find location of yarn cache
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - uses: actions/cache@v1
+        with:
+          path: ${{ steps.yarn-cache.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
```

Comment:
i assume the glob is due to nohoist packages?

Reply:
Dunno tbh, I'm assuming this is a thing that's supposed to work in any repo and probably from when we had the `frontend/` folder. Not touching it here though :grin: